### ISSUE TITLE
a small question about the loss function

### ISSUE THREAD

**Issue by PB20000090** [lucidrains/audiolm-pytorch#190]
In soundstream.py line 58:
def hinge_gen_loss(fake)
      return -fake.mean(）
May I ask if this should return (F.relu(1 - fake)).mean()?
Thanks

**Comment by lucidrains** [comment]
so i'm taking the formulation used in stylegan2, which in turn uses [this paper](https://arxiv.org/abs/1705.02894) [(paperswithcode explanation)](https://paperswithcode.com/method/gan-hinge-loss)
